reconcile/glitchtip/README.md

```diff
@@ -0,0 +1,148 @@
+<font size=24> GlitchTip </font>
```

Comment:
I assume the idea is to have a domain-specific module. I find this idea very interesting. How would this look for utils that are shared? Have you planned on restructuring the modules?

The current structure would suggest moving reconciler.py into `utils`. I would prefer to see it that way.

Reply:
The idea was the following:
* `reconcile/glitchtip` is the integration and everything which belongs to the integration, e.g., `reconciler.py` should not be used from everywhere else, and therefore, it isn't in `utils`
* `reconcile/utils/glitchtip` is independent of the integration and can be used by other integrations/utils. 
* In general I like submodules instead of bloated python files :)

Reply:
I am 100% with you. Just afraid of introducing another standard without considering the other modules and how it would look for them, which could cause this to be a beautiful snowflake :)